content/docs/reference/routes/headers.mdx

```diff
@@ -185,8 +190,8 @@ Pass ID and Access tokens as dynamic headers in the request:
           - email:
               is: user@example.com
   set_request_headers:
-    x-pomerium-idp-id-token: $pomerium.id_token
-    x-pomerium-idp-access-token: $pomerium.access_token
+    x-pomerium-idp-id-token: ${pomerium.id_token}
+    x-pomerium-idp-access-token: ${pomerium.access_token}
```

Comment:
Just a suggestion: we could include `${pomerium.client_cert_fingerprint}` in the example.

Reply:
Done.